glide.yaml

```diff
@@ -20,3 +20,5 @@ import:
 - package: github.com/arschles/kubeapp
   subpackages:
   - api
+- package: github.com/docker/docker
+  version: 2b27fe17a1b3fb8472fde96d768fa70996adf201
```

Comment:
Yes... it was previously failing trying to set docker/docker to 0f5c9d301b9b1cca66b3ea0f9dec3b5317d3686d.

The interesting thing here...

You've met success by doing `glide up`.  `make bootstrap` normally does a `glide install`, which uses the precise versions called out in the `glide.lock`, vs. what `glide up` does-- which is to update to the latest version of everything (within whatever constraints are described in `glide.yaml`). So you've stepped outside the typical workflow, so to speak, that would be followed for someone who had just cloned this repo for the first time (as I had).  I think one way or another we've got to get that fixed.